test/steps/kogitoapp.go

```diff
@@ -51,15 +50,16 @@ func registerKogitoAppSteps(s *godog.Suite, data *Data) {
 
 // Deploy service steps
 func (data *Data) deployQuarkusExampleServiceWithNative(contextDir, native string) error {
-	return framework.DeployExample(data.Namespace, framework.GetDefaultInstallerType(),
+	return framework.DeployService(data.Namespace, framework.GetDefaultInstallerType(),
 		framework.KogitoAppDeployment{
-			AppName:     filepath.Base(contextDir),
-			ContextDir:  contextDir,
-			Runtime:     v1alpha1.QuarkusRuntimeType,
-			Native:      framework.MustParseEnabledDisabled(native),
-			Persistence: false,
-			Events:      false,
-			Labels:      nil,
+			AppName:      filepath.Base(contextDir),
+			GitSourceURI: config.GetExamplesRepositoryURI(),
```

Comment:
missing `GitSourceReference: config.GetExamplesRepositoryRef` or ?

Reply:
yes, I forgot to add it there